File: Lab_Work/CPP/Batch_2/day_12/EmployeeVector/Address.cpp
#include <iostream>
#include "address_header.h"

Address::Address(const string &area, const string &city, const string &pincode) : area(area), city(city), pincode(pincode) {}

void Address::displayAddress() const
{
	cout << area << ", " << city << ", " << pincode;
}

File: Lab_Work/Java/practices/src/eclipse_workspace/day7/emp/Address.java
package eclipse_workspace.day7.emp;

public class Address {
	private String street;
	private String city;
	private String state;
	private String country;
	private int pincode;

	public Address(String street, String city, String state, String country, int pincode) {
		this.street = street;
		this.city = city;
		this.state = state;
		this.country= country;
		this.pincode = pincode;
	}

	@Override
	public String toString() {
		return "Address{" +
				"street='" + street + '\'' +
				", city='" + city + '\'' +
				", state='" + state + '\'' +
				", country='" + country + '\'' +
				", pincode=" + pincode +
				'}';
	}
}
